projectile.el

```diff
@@ -4529,6 +4529,8 @@ See `projectile--cleanup-known-projects'."
   "Clear both `projectile-known-projects' and `projectile-known-projects-file'."
   (interactive)
   (setq projectile-known-projects nil)
+  (when projectile-auto-discover
+    (projectile-discover-projects-in-search-path))
```

Comment:
I see that this command internally updates the known projects, so the second invocation is redundant. Probably you should use `if-else` instead of a `when`.